zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -18,23 +18,25 @@ ZULIP_MESSAGE_TEMPLATE = "**{username}**: {message}"
 SLACK_MESSAGE_TEMPLATE = "<{username}> {message}"
 
 
-def check_zulip_message_validity(
+def get_zulip_message_slack_channel(
     msg: Dict[str, Any], zulip_to_slack_map: Dict[str, Any], bot_email: str
-) -> Tuple[bool, str]:
+) -> Union[str, None]:
     is_a_stream = msg["type"] == "stream"
     if not is_a_stream:
-        return False, ""
+        return None
 
     stream_name = msg["display_recipient"]
     topic_name = msg["subject"]
-    stream_topic = stream_name + "-" + topic_name
+    stream_topic = f"{stream_name}-{topic_name}"
```

Comment:
The f-string formatting is an improvement, but reading this I wondered if you'd considered using a stream-topic tuple in your dicts instead?

Reply:
Converted to tuple.